### ISSUE TITLE
[Bug] key control event "activateFile" is not keyboard accessible - runs off mouse click event

### ISSUE THREAD

**Issue by lemonbob** [pqina/filepond#831]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you updated FilePond and its plugins?

- [X] I have updated FilePond and its plugins

### Describe the bug

The event activateFile runs off a mouse click event handler in filepond (Vue plugin generates it as an emitted event) and is a key control event. 

As per WCAG 2.1 AA guidelines, to be accessible key controllable clickable items should be mapped to the keyboard (e.g. keydown/keyup enter and arrow keys) and be tabbable elements, otherwise non-mouse users are denied access to key functionality.

### Reproduction

Cannot tab onto or generate activateFile events on file-pond file elements via keyboard.

### Environment

```markdown
- Device:all
- OS:n/a
- Browser:all
```
